hack/format-all-files.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -e
+
+hack_dir="$(dirname $0)"
+repo_dir="$(dirname $hack_dir)"
+
+find . -iname "*.go" ! -wholename "./vendor/*" -print0 | \
```

Comment:
@rene-dekker points out that we need an exclusion for the `third-party` folder too